src/System.Windows.Forms/src/System/Windows/Forms/TabControl.ControlCollection.cs

```diff
@@ -19,11 +17,11 @@ public ControlCollection(TabControl owner) : base(owner)
                 _owner = owner;
             }
 
-            public override void Add(Control value)
+            public override void Add(Control? value)
             {
-                if (!(value is TabPage))
+                if (value is not TabPage)
```

Comment:
```suggestion
                if (value is not TabPage tabPage)
```
and delete line 27? The same pattern can be found on line 70.